src/System.Private.ServiceModel/tests/Scenarios/Contract/Message/MessageContractCommon.4.1.0.cs

```diff
@@ -78,7 +76,7 @@ public interface IMessageContract
 
     public class MessageContractTypes
     {
-        [MessageContract(IsWrapped = true, WrapperName = "CustomWrapperName", WrapperNamespace = "http://www.contoso.com")]
```

Comment:
[No change required -- observation only] It's too bad the original test didn't use the same constants in MessageContractConstants for these attribute named parameters.  It would have eliminated so much string duplication scattered in the contracts and tests.